assets/js/push.js

```diff
@@ -580,7 +581,19 @@ jQuery( window ).on( 'load', () => {
 				deleteNode.parentNode.removeChild( deleteNode );
 
 				delete selectedConnections[ type + id ];
-
+				selectedText = sprintf(
+					/* translators: 1) Selected connection content singular name. 2) Selected connection content plural name. */
```

Comment:
```suggestion
					/* translators: %d: Number of selected connections. */
```

For `_n` translator notes are a little odd, the comment applies to both strings as the placeholders need to be identical.